docs/release-notes/8.4.asciidoc

```diff
@@ -0,0 +1,74 @@
+* Fixes a performance issue with alerts that have large fields ({pull}135956[#135956]).
+* Replaces the response actions timeline with the response actions log ({pull}135360[#135360]).
+* Updates the UI for displaying Exceptions ({pull}135255[#135255]).
+* Fixes performance issues with rules management ({pull}135311[#135311]).
+* Allows you to define a fallback `@timestamp` when you've defined a timestamp override ({pull}135116[#135116]).
+* Enhances the Host risk score modal UI ({pull}133708[#133708]).
+* Updates the lists index template to new logic ({pull}133067[#133067]).
+* Adds event filters to the Event Correlation rule type ({pull}132507[#132507]).
+* Allows you to define a data view or index pattern for rules ({pull}130929[#130929]).
+* Adds a treemap and multi-dimensional alert grouping to the Alerts page ({pull}126896[#126896]).
+
+* Fixes an incorrect counter for exported rules ({pull}138598[#138598]).
+* Fixes event filters based on OS version ({pull}138517[#138517]).
+* Fixes a bug that could change the batch size for event search in Indicator Rules ({pull}138356[#138356]).
+* Fixes a bug that could crash the alert flyout ({pull}138331[#138331]).
+* Fixes the preview button for {ml} rules ({pull}137878[#137878]).
+* Fixes a bug that could crash the Endpoint list when a policy ID was missing ({pull}137788[#137788]).
+* Fixes a bug that could interfere with opening host or user details pages ({pull}137719[#137719]).
+* Fixes several bugs related to refreshing the Alerts page ({pull}137620[#137620]).
+* Fixes a bug with bulk rule deletion with a defined data view ({pull}137585[#137585]).
+* Fixes a bug with look-back time in advanced query previews ({pull}137517[#137517]).
+* Fixes a bug that prevented Threshold Rules' timeline templates from being respected during investigations ({pull}137233[#137233]).
+* Fixes a permissions bug related to the Save Timeline button ({pull}136724[#136724]).
+* Fixes a bug with selecting timeline templates with the same name ({pull}135694[#135694]).
+* Fixes field aliases to `signal-threshold_result.*` ({pull}135565[#135565]).
+* Fixes a bug that lost track of which rules you had selected after refreshing the Rules page ({pull}135533[#135533]).
+* Fixes a bug that lost track of which rules you had selected alter applying a bulk action on the Rules page ({pull}135291[#135291]).
+* Fixes a bug that prevented the rule details table from pausing auto-refresh while bulk actions are being applied ({pull}135208[135208]).
+* Fixes a bug that could cuase queries with nested fields to fail open ({pull}134866[#134866]).
+* Fixes a bug that slowed down the display of Network details ({pull}133539[#133539]).
+* Various minor bug fixes and enhancements ({pull}133079[#133079], {pull}138135[#138135], {pull}138286[#138286], {pull}138131[#138131], {pull}137588[#137588], {pull}137511[#137511], {pull}137492[#137492], {pull}135907[#135907], {pull}135426[#135426]).
```

Comment:
Might be good to list these separately for visibility. Though, if you were specifically asked not to, ignore this comment. :)